libraries/AP_RCProtocol/AP_RCProtocol_CRSF.cpp

```diff
@@ -80,10 +80,11 @@
 
 extern const AP_HAL::HAL& hal;
 
-// #define CRSF_DEBUG
+//#define CRSF_DEBUG
+#define CRSF_DEBUG_CHARS
```

Comment:
Need to turn off this

Reply:
it's always nested inside CRSF_DEBUG, but I'll turn it off anyway